### ISSUE TITLE
an evaluation question about the tutorial on motr

### ISSUE THREAD

**Issue by ZhiCheng-H** [ifzhang/ByteTrack#194]
Hello, thanks for your wonderful work, I'm trying to do an evaluation following the [https://github.com/ifzhang/ByteTrack/tree/main/tutorials/motr](url), The step1 and step2 has successfully completed. 
But when I run the eval code mentioned in the readme.md.
`python3 eval.py \
    --meta_arch motr \
    --dataset_file e2e_joint \
    --epoch 200 \
    --with_box_refine \
    --lr_drop 100 \
    --lr 2e-4 \
    --lr_backbone 2e-5 \
    --pretrained exps/e2e_motr_r50_mot17val/motr_final.pth \
    --output_dir exps/e2e_motr_r50_mot17val \
    --batch_size 1 \
    --sample_mode 'random_interval' \
    --sample_interval 10 \
    --sampler_steps 50 90 120 \
    --sampler_lengths 2 3 4 5 \
    --update_query_pos \
    --merger_dropout 0 \
    --dropout 0 \
    --random_drop 0.1 \
    --fp_ratio 0.3 \
    --query_interaction_layer 'QIM' \
    --extra_track_attn \
    --mot_path ./MOT17/images/train
    --data_txt_path_train ./datasets/data_path/mot17.half \
    --data_txt_path_val ./datasets/data_path/mot17.val \
    --resume model_final.pth \`

I received a problem`Segmentation fault (core dumped)`. I'm very confused that why does this problem arise and how to solve it.
